mediator/med_phases_prep_ice_mod.F90

```diff
@@ -53,16 +50,12 @@ subroutine med_phases_prep_ice(gcomp, rc)
     ! local variables
     type(ESMF_StateItem_Flag)      :: itemType
     type(InternalState)            :: is_local
+    type(ESMF_Field)               :: lfield
     integer                        :: i,n,n1,ncnt
     character(len=CS)              :: fldname
     integer                        :: fldnum
     integer                        :: mapindex
     real(R8), pointer              :: dataptr(:)
```

Comment:
init with null()

Reply:
There are also other cases that pointiest are not set to null() but I did not mark all of them.